test/__mocks__/session.mock.ts

```diff
@@ -864,6 +868,18 @@ export const TRUST_PARTIAL_DATE: Trust = {
   unable_to_obtain_all_trust_info: "No"
 };
 
+export const OVER_SEAS_ENTITY_MOCK_DATA: ICompanyDetails = {
+  companyName: "Polish brewery",
```

Comment:
replace Polish brewery by "acme" (should not be calling out some country in company name)